code/modules/mob/living/say.dm

```diff
@@ -39,21 +39,21 @@ var/list/department_radio_keys = list(
 
 	  //kinda localization -- rastaf0
 	  //same keys as above, but on russian keyboard layout. This file uses cp1251 as encoding.
-	  ":�" = "right hand",	"#�" = "right hand",	".�" = "right hand",
-	  ":�" = "left hand",	"#�" = "left hand",		".�" = "left hand",
-	  ":�" = "intercom",	"#�" = "intercom",		".�" = "intercom",
-	  ":�" = "department",	"#�" = "department",	".�" = "department",
-	  ":�" = "Command",		"#�" = "Command",		".�" = "Command",
-	  ":�" = "Science",		"#�" = "Science",		".�" = "Science",
-	  ":�" = "Medical",		"#�" = "Medical",		".�" = "Medical",
-	  ":�" = "Engineering",	"#�" = "Engineering",	".�" = "Engineering",
-	  ":�" = "Security",	"#�" = "Security",		".�" = "Security",
-	  ":�" = "whisper",		"#�" = "whisper",		".�" = "whisper",
-	  ":�" = "binary",		"#�" = "binary",		".�" = "binary",
-	  ":�" = "alientalk",	"#�" = "alientalk",		".�" = "alientalk",
-	  ":�" = "Syndicate",	"#�" = "Syndicate",		".�" = "Syndicate",
-	  ":�" = "Supply",		"#�" = "Supply",		".�" = "Supply",
-	  ":�" = "changeling",	"#�" = "changeling",	".�" = "changeling"
+	  ":ê" = "right hand",	"#ê" = "right hand",	".ê" = "right hand",
+	  ":ä" = "left hand",	"#ä" = "left hand",		".ä" = "left hand",
+	  ":ø" = "intercom",	"#ø" = "intercom",		".ø" = "intercom",
+	  ":ð" = "department",	"#ð" = "department",	".ð" = "department",
+	  ":ñ" = "Command",		"#ñ" = "Command",		".ñ" = "Command",
+	  ":ò" = "Science",		"#ò" = "Science",		".ò" = "Science",
+	  ":ü" = "Medical",		"#ü" = "Medical",		".ü" = "Medical",
+	  ":ó" = "Engineering",	"#ó" = "Engineering",	".ó" = "Engineering",
+	  ":û" = "Security",	"#û" = "Security",		".û" = "Security",
+	  ":ö" = "whisper",		"#ö" = "whisper",		".ö" = "whisper",
+	  ":è" = "binary",		"#è" = "binary",		".è" = "binary",
+	  ":ô" = "alientalk",	"#ô" = "alientalk",		".ô" = "alientalk",
+	  ":å" = "Syndicate",	"#å" = "Syndicate",		".å" = "Syndicate",
+	  ":é" = "Supply",		"#é" = "Supply",		".é" = "Supply",
+	  ":ï" = "changeling",	"#ï" = "changeling",	".ï" = "changeling"

```

Comment:
I have no idea why robustin gave this code any attention but good man rastaf0 did great job at making comms available for us for free